tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["remix.env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["remix.env.d.ts", "**/*.ts", "**/*.tsx", "app/components/Footer.tsx"],
```

Comment:
The `**/*.tsx` pattern should already cover the Footer. It means anything with the `.tsx` extension will be compiled by TypeScript.

Reply:
Thank you for reminding me of that, I completely forgot to remove that 😅